abi/abi.go

```diff
@@ -1,33 +1,66 @@
 package abi
 
 import (
-	"fmt"
-	"math"
-
-	"github.com/algorand/go-algorand/data/abi"
+	avm_abi "github.com/algorand/avm-abi/abi"
```

Comment:
@jasonpaulos For my understanding:  Is there a reason to prefer `avm_abi` here while https://github.com/algorand/go-algorand/pull/4375 uses `abi`?

Reply:
Disregard - There's a different package structure here than go-algorand.  Resolving.